src/main/java/org/jahia/modules/sitemap/utils/VanityUrls.java

```diff
@@ -55,6 +56,10 @@ public static String getActiveUrl(JCRNodeWrapper node, String langCode) throws R
         return getActiveUrl(vanityUrls);
     }
 
+    public static boolean canUseVanity() {
+        return Boolean.parseBoolean(SettingsBean.getInstance().getString("urlRewriteSeoRulesEnabled", "false"));
```

Comment:
I think there's an equivalent `isUrlRewriteSeoRulesEnabled()` to avoid the boolean conversion but this works as well.

Reply:
Yes, you're right there is